frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -1131,16 +1131,17 @@ export default ({
       }
     },
     processSwitchQueue: debounce(async function () {
```

Comment:
(If it's decided that the queue is not necessary)
The name of this function should be changed too. (probably something like `processChatroomSwitch`)